weston-ivi-shell/src/ivi-controller.c

```diff
@@ -2204,7 +2204,7 @@ int
 setup_ivi_controller_server(struct weston_compositor *compositor,
                             struct ivishell *shell)
 {
-    if (wl_global_create(compositor->wl_display, &ivi_wm_interface, 1,
+    if (wl_global_create(compositor->wl_display, &ivi_wm_interface, ivi_wm_interface.version,
```

Comment:
@efriedrich, could you please give the detail comment here? something relate to keep the hard code + tests